docs/Getting Started Part 2/index.md

```diff
@@ -0,0 +1,323 @@
+# Getting Started Part 2
+
+## Connecting HMI to SDL
+
+WebSocket is the primary means of communicating with the SDL Core component from the vehicle. In a basic example, an HTML5 HMI would use a native WebSocket library to communicate with SDL Core.
+
+The HMI Adapter must:
+
+!!! must
+  * be installed on the same vehicle HU OS where SDL is installed, or the HMI must be able to be networked to SDL and address it via a static IP address.
+  * create and initialize components which are defined in the HMI_API specification for the version of SDL which is running on the vehicle HU. (BasicCommunication, UI, Buttons, VR, TTS, Navigation, VehicleInfo, RC, AppService)
+  * Establish a separate WebSocket connection with SDL for each of components defined in the HMI_API specification.
+  * Use the appropriate corresponding connection when sending responses and notifications to any connected component.
+
+!!!
+
+## Handshake
+For opening a WebSocket connection, a handshake must be performed.
+
+!!! info
+  1. Client/Server relationship
+    * SDL Core is the Server
+    * The HMI is the Client
+  2. Host
+    * SDL Core is listening on **127.0.0.1:8087** by default
+    * The IP and port is configurable in SDL Core's smartDeviceLink.ini file
+  3. WebSocket Protocol Version 13 is used by SDL Core
+
+!!!
+
+## Component Registration
+
+### Request
+
+The HMI must register each component which can communicate with SDL Core using the following RPC format
+
+| Key     | Value Info    |
+| :------------- | :------------- |
+| "id"       | a multiple of 100 (100, 200, 300, ...) |
+| "jsonrpc" | "2.0" - constant for all messages between SDL Core and the HMI|
+| "method" | "MB.registerComponent" - the request is assigned to SDL Core's MessageBroker where the component name will be associated with the socket ID. Further, SDL Core will send messages related to the named component over the corresponding connection|
+|"componentName"| The name of the component being registered. Must correspond to the appropriate component name described in the current guidelines.|
+
+Example Request:
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 100,
+  "method": "MB.registerComponent",
+  "params": {
+    "componentName": "BasicCommunication"
+  }
+}
+```
+
+The possible componentNames are:  
+  * `BasicCommunication` - Generic interface responsible for a collection of information and user actions. Includes `UpdateAppList` and `OnAppRegistered`.
```

Comment:
this bug annoyed me too, the list html only generates if the line above the list is empty